core/src/test/java/org/projectnessie/cel/interpreter/ActivationTest.java

```diff
@@ -50,8 +49,8 @@ void activation() {
   @Test
   void resolve() {
     Activation activation = newActivation(Collections.singletonMap("a", True));
-    assertThat(activation.resolveName("a").a).isSameAs(True);
-    assertThat(activation.resolveName("a").b).isSameAs(true);
+    assertThat(activation.resolveName("a").present()).isSameAs(true);
```

Comment:
Nit: use `.isTrue()`, because the result's a primitive `boolean`
(no need to change the PR though)